code/lib/cli/src/sandbox-templates.ts

```diff
@@ -92,7 +92,10 @@ type BaseTemplates = Template & {
 const baseTemplates = {
   'cra/default-js': {
     name: 'Create React App Latest (Webpack | JavaScript)',
-    script: 'npx create-react-app {{beforeDir}}',
+    script: `
+      npx create-react-app {{beforeDir}} && \
+      jq '.browserslist.production[0] = ">0.9%"' package.json > tmp.json && mv tmp.json package.json
```

Comment:
I am unsure whether this will work in everyone's machines. This might work in CI because we have `jq` there, but for any contributor it could break in case they don't have the binary

Reply:
`jq` was already used in this file